homeassistant/components/switch/esphome.py

```diff
@@ -47,6 +47,11 @@ def icon(self) -> str:
         """Return the icon."""
         return self._static_info.icon
 
+    @property
+    def assumed_state(self) -> bool:
+        """Return true if we do optimistic updates."""
+        return self._static_info.optimistic
```

Comment:
Switches can also be optimistic in ESPHome (for example template switches without a state lambda). Forgot to include it in the original PR.